lighthouse-docker/Dockerfile.headful

```diff
@@ -0,0 +1,60 @@
+FROM node:9-slim
```

Comment:
could we get some composition going and share most of this? :)


```dockerfile
FROM lighthouse_base

# do headless vs. headful things
...

Reply:
Yea good idea. I hadn't planned to maintain the headful version but the xvfb stuff is such a pain that it was worth keeping around to provide an example.